### ISSUE TITLE
Display a summary in the Atom entry with ajax.

### ISSUE THREAD

**Issue by corestrike** [infoScoop/infoscoop#164]
&lt;b&gt;What steps will reproduce the problem?&lt;/b&gt;
&lt;b&gt;1.&lt;/b&gt;
&lt;b&gt;2.&lt;/b&gt;
&lt;b&gt;3.&lt;/b&gt;

&lt;b&gt;What is the expected output? What do you see instead?&lt;/b&gt;

&lt;b&gt;Please use labels and text to provide additional information.&lt;/b&gt;
